File: TransCoder_tokenized_test_set_functions/cpp/REMOVE_MINIMUM_NUMBER_ELEMENTS_NO_COMMON_ELEMENT_EXIST_ARRAY.cpp
int minRemove ( int a [ ], int b [ ], int n, int m ) {
  unordered_map < int, int > countA, countB;
  for ( int i = 0;
  i < n;
  i ++ ) countA [ a [ i ] ] ++;
  for ( int i = 0;
  i < m;
  i ++ ) countB [ b [ i ] ] ++;
  int res = 0;
  for ( auto x : countA ) if ( countB . find ( x . first ) != countB . end ( ) ) res += min ( x . second, countB [ x . first ] );
  return res;
}

File: TransCoder_tokenized_test_set_functions/java/REMOVE_MINIMUM_NUMBER_ELEMENTS_NO_COMMON_ELEMENT_EXIST_ARRAY.java
public static int minRemove ( int a [ ], int b [ ], int n, int m ) {
  HashMap < Integer, Integer > countA = new HashMap < Integer, Integer > ( );
  HashMap < Integer, Integer > countB = new HashMap < Integer, Integer > ( );
  for ( int i = 0;
  i < n;
  i ++ ) {
    if ( countA . containsKey ( a [ i ] ) ) countA . put ( a [ i ], countA . get ( a [ i ] ) + 1 );
    else countA . put ( a [ i ], 1 );
  }
  for ( int i = 0;
  i < m;
  i ++ ) {
    if ( countB . containsKey ( b [ i ] ) ) countB . put ( b [ i ], countB . get ( b [ i ] ) + 1 );
    else countB . put ( b [ i ], 1 );
  }
  int res = 0;
  Set < Integer > s = countA . keySet ( );
  for ( int x : s ) if ( countB . containsKey ( x ) ) res += Math . min ( countB . get ( x ), countA . get ( x ) );
  return res;
}